conf/application.conf

```diff
@@ -151,6 +151,7 @@ feature-switch {
     enabled = true
     released-in-production = false
   }
+  isPOARelevantAmount.enabled = true
```

Comment:
Consider removing. Defailt is true anyway, so there will be one less thing to clean up when the temporary switch is removed.